src/SOS/Strike/sosdocsunix.txt

```diff
@@ -32,16 +32,17 @@ DumpHeap (dumpheap)                EEStack (eestack)
 DumpVC                             ClrStack (clrstack)
 FinalizeQueue (finalizequeue)      GCInfo
 GCRoot (gcroot)                    EHInfo
-PrintException (pe)                bpmd (bpmd)
+ObjSize                            bpmd (bpmd)
```

Comment:
Can you put `(objsize)` after ObjSize?  This indicates that on lldb/dotnet-dump that the lowercase alias is available.